composer.json

```diff
@@ -9,6 +9,11 @@
       "name": "jarednova"
     }
   ],
+  "autoload": {
+    "psr-4": {
+      "": "src/"
```

Comment:
Please encourage user to always have namespaces.

[_source_](https://github.com/szepeviktor/starter-plugin#what-to-avoid)

Reply:
You’re right.

But I fear that adding an example namespace would lead to developers not updating it. I’d rather add a small hint about this in the README file.

I updated the PR title, description and files accordingly.

Reply:
> lead to developers not updating it

One of my Gmail related documents contains my email address. Users register with **my** email address 🤯

I use `Company\WordPress\PluginName` as an example namespace.
https://github.com/szepeviktor/starter-plugin/blob/9a1a8ffb5d2ef35266fd0f0c2bbda9ce931b6471/plugin-name.php#L28C11-L28C39